app/scripts/controllers/incoming-transactions.test.js

```diff
@@ -2,7 +2,7 @@ import { strict as assert } from 'assert';
 import sinon from 'sinon';
 import proxyquire from 'proxyquire';
 import nock from 'nock';
-import { cloneDeep } from 'lodash';
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
Why are we changing this. I think we should keep it the same 
```suggestion
import { cloneDeep } from 'lodash';```

Reply:
:cookie this is to reduce the bundle size, you can refer to this article => https://www.blazemeter.com/blog/import-lodash-libraries#:~:text=If%20you%20want%20to%20minimize,function%20by%20its%20own%20module.